app/fastlane/Fastfile

```
@@ -375,40 +395,69 @@ platform :android do
+              begin
+                # Try passing the credentials file content as json_key_data
+                creds_content = File.read(adc_creds_path)
+                upload_options[:json_key_data] = creds_content
+              rescue => e
+                UI.error("Failed to read ADC credentials: #{e.message}")
+                # Fallback: let supply try to use ADC automatically
+                UI.message("🔄 Falling back to automatic ADC detection")
+              end
+            else
+              UI.error("❌ ADC credentials not found at: #{adc_creds_path}")
+            end
```

RuboCop (1.80.2)

[convention] Avoid more than 3 levels of block nesting.

(Metrics/BlockNesting)